packages/manager/apps/dedicated/client/app/dedicated/server/server.service.js

```diff
@@ -12,257 +12,237 @@ import snakeCase from 'lodash/snakeCase';
 import sortBy from 'lodash/sortBy';
 import uniq from 'lodash/uniq';
 
-angular
-  .module('services')
-  .constant('SERVERSTATS_PERIOD_ENUM', {
-    HOURLY: {
-      standardDays: 0,
-      standardHours: 0,
-      standardMinutes: 1,
-      standardSeconds: 60,
-      millis: 60000,
-    },
-    DAILY: {
-      standardDays: 0,
-      standardHours: 0,
-      standardMinutes: 5,
-      standardSeconds: 300,
-      millis: 300000,
-    },
-    WEEKLY: {
-      standardDays: 0,
-      standardHours: 0,
-      standardMinutes: 30,
-      standardSeconds: 1800,
-      millis: 1800000,
-    },
-    MONTHLY: {
-      standardDays: 0,
-      standardHours: 2,
-      standardMinutes: 120,
-      standardSeconds: 7200,
-      millis: 7200000,
-    },
-    YEARLY: {
-      standardDays: 1,
-      standardHours: 24,
-      standardMinutes: 1440,
-      standardSeconds: 86400,
-      millis: 86400000,
-    },
-  })
-  .constant('HARDWARE_RAID_RULE_DEFAULT_NAME', 'managerHardRaid')
-  .service('Server', function serverF(
-    $http,
-    $q,
-    constants,
-    coreConfig,
-    $cacheFactory,
-    $rootScope,
-    $translate,
-    WucApi,
-    Polling,
-    OvhApiOrder,
-    OvhApiOrderCatalogPublic,
-    OvhHttp,
-    SERVERSTATS_PERIOD_ENUM,
-    HARDWARE_RAID_RULE_DEFAULT_NAME,
-  ) {
-    this.OvhApiOrderCatalogPublic = OvhApiOrderCatalogPublic;
-    this.OvhApiOrderBaremetalPublicBW = OvhApiOrder.Upgrade()
-      .BaremetalPublicBandwidth()
-      .v6();
-    this.OvhApiOrderBaremetalPrivateBW = OvhApiOrder.Upgrade()
-      .BaremetalPrivateBandwidth()
-      .v6();
-
-    const self = this;
-    const serverCaches = {
-      ipmi: $cacheFactory('UNIVERS_DEDICATED_SERVER_IPMI'),
-      ovhTemplates: $cacheFactory(
-        'UNIVERS_DEDICATED_SERVER_INSTALLATION_OVH_TEMPLATE',
-      ),
-    };
-    const requests = {
-      serverDetails: null,
-      serverStatistics: null,
-      serverStatisticsConst: null,
-    };
-    const path = {
-      product: 'dedicated/server/{serviceName}',
-      blocks: 'module/ip/blocks',
-      installation: 'installation',
-      installationMe: 'me/installationTemplate',
-      ip: 'ip',
-      sshMe: 'me/sshKey',
-      order: 'order/dedicated/server/{serviceName}',
-      sms: 'sms',
-      proxy: 'apiv6',
-      dedicatedServer: 'apiv6/dedicated/server',
-    };
-
-    // TODO delete this
-    const serverCache = $cacheFactory('UNIVERS_DEDICATED_SERVER_');
-
-    function resetCache(targetedCache) {
-      if (targetedCache && targetedCache !== serverCaches.all) {
+angular.module('services').service(
+  'Server',
+  class ServerF {
```

Comment:
```suggestion
  export default class ServerF {
```
and declare the `Server` service in some `index.js` or `server.module.js`